src/app/components/Form/FormFieldWrapper.tsx

```diff
@@ -4,16 +4,17 @@ import { WrappedFieldProps } from 'redux-form';
 
 export interface FormFieldWrapperProps extends WrappedFieldProps{
     label:string;
+    controlId?:string;
 }
 
 export default class FormFieldWrapper extends React.PureComponent<FormFieldWrapperProps>{
     render(){
-        const {meta:{touched, error, warning}, label} = this.props;
+        const {meta:{touched, error, warning}, label, controlId} = this.props;
         const {children} = this.props;
         const validationState = touched ? (error ? 'error': (warning ? 'warning': undefined)) : undefined;
         
         return (
-            <FormGroup validationState={validationState}>
+            <FormGroup validationState={validationState} controlId={controlId}>
```

Comment:
I don't see the controlId being used anywhere but if we need one could we perhaps use something like 
```ts
`${formname}-${fieldName}
```

Reply:
We don't need it at this point; I forgot I had added that in - it has been removed